packages/blueprints/src/studio0/index.ts

```diff
@@ -1,24 +1,55 @@
-import { BlueprintManifestType, StudioBlueprintManifest } from '@sofie-automation/blueprints-integration'
-import { studioConfigManifest } from './config-manifests'
+import {
+	BlueprintManifestType,
+	JSONBlobStringify,
+	StudioBlueprintManifest,
+} from '@sofie-automation/blueprints-integration'
 import { getBaseline } from './getBaseline'
 import { getShowStyleId } from './getShowStyleId'
-import { studioMigrations } from './migrations'
+// import { studioMigrations } from './migrations'
+import ConfigSchema = require('./config-schema.json')
+import DefaultMappings, { AtemMappings } from './migrations/mappings-defaults'
+import { StudioConfig } from '../generated/studio-config'
 
 declare const VERSION: string // Injected by webpack
 declare const VERSION_TSR: string // Injected by webpack
 declare const VERSION_INTEGRATION: string // Injected by webpack
 declare const TRANSLATION_BUNDLES: string // injected by webpack
 
-const manifest: StudioBlueprintManifest = {
+const manifest: StudioBlueprintManifest<StudioConfig> = {
 	blueprintType: BlueprintManifestType.STUDIO,
 
 	blueprintId: 'sofie-generic-studio',
 	blueprintVersion: VERSION,
 	integrationVersion: VERSION_INTEGRATION,
 	TSRVersion: VERSION_TSR,
 
-	studioConfigManifest,
-	studioMigrations,
+	studioConfigSchema: JSONBlobStringify(ConfigSchema),
+	configPresets: {
```

Comment:
I think these config presets & playoutDevice configs could have better defaults to make it easier for new users to demo Sofie